docs/modelserving/v1beta1/llm/huggingface/multi-node/README.md

```diff
@@ -52,6 +52,29 @@ You can set StartupProbe in ServingRuntime for your own situation.
 ..
 ~~~
 
+Multi-node setups typically use the `RollingUpdate strategy`, which ensures that the existing service remains operational until the new service becomes Ready. However, this approach requires more than twice the resources to function effectively. Therefore, during the development phase, it is more appropriate to use the `Recreate strategy`.
+
+~~~
+spec:
+  predictor:
+    deploymentStrategy:
+      type: Recreate
+    model:
+      modelFormat:
+        name: huggingface
+      runtime: kserve-huggingfaceserver-multinode
+      storageUri: pvc://XXXX
+    workerSpec: {}
+~~~
+Additionally, modifying the PipelineParallelSize (either increasing or decreasing it) can impact the existing service due to the default behavior of the Deployment resource. It is important to note that **PipelineParallelSize is not an autoscaling concept**; rather, it determines how many nodes will be used to balance the model. For this reason, it is strongly recommended not to modify this setting in production environments.
```

Comment:
Small suggestion, maybe?

```suggestion
Additionally, modifying the PipelineParallelSize (either increasing or decreasing it) can impact the existing service due to the default behavior of the Deployment resource. It is important to note that **PipelineParallelSize is not an autoscaling concept**; rather, it determines how many nodes will be used to run the model. For this reason, it is strongly recommended not to modify this setting in production environments.
```

For load-balancing, I'd assume using replicas with single-node would be enough.